web/src/demo_components/button.dart

```diff
@@ -78,7 +78,7 @@ class _$ButtonProps extends UiProps {
 class _$ButtonState extends UiState {}
 
 @Component()
-class ButtonComponent<T extends ButtonProps, S extends ButtonState> extends UiStatefulComponent<T, S> {
+class ButtonComponent<T extends ButtonProps, S extends ButtonState> extends UiStatefulComponent2<T, S> {
```

Comment:
Should we make a copy of all the demo components so that we have version 1 and version 2 components to smoke test as we work our way through 3.0.0?

Reply:
Done!